app.yaml

```diff
@@ -0,0 +1,27 @@
+runtime: nodejs10
+ 
+handlers:
```

Comment:
I think we might be able to trim this to something smaller like this:

```
handlers:
- url: /.*
  static_files: dist/web/\1
  upload: dist/web/.*
```

Reply:
The rest of the file is trimmed and the app.yaml file is updated.